### PR TITLE
SPDX license field

### PR DESCRIPTION
**Pull request by isaacs**
Use `BSD-2-Clause` instead of just `BSD`

### CODE REVIEW

**Comment by ForbesLindesay** [comment]
Do we also need to update the LICENSE file?

**Comment by isaacs** [comment]
Nah, I just used the SPDX-compatible designator for what's in the LICENSE file.

I've moved all my own stuff to ISC, which is a bit more clear imo (especially since there's only one of them, unlike the many BSD's) but legally speaking they're equivalent.

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "type": "git",
     "url": "https://github.com/ForbesLindesay/sha.git"
   },
-  "license": "BSD",
+  "license": "BSD-2-Clause",
```
Comment (KenanY) [inline package.json]: I think this should be:

```
{ "license" : "(BSD-2-Clause OR MIT)" }
```

since there's a dual-license at the moment.

Comment (isaacs) [inline package.json]: Ah, indeed!  I didn't read far enough down the file ;)

**Pull request merged**